modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/AnnotationsBasedDescriber.java

```diff
@@ -343,6 +354,25 @@ private void declareOperationParameters(Method method, OperationDescriptor opera
             {
                 operation.withModelProperty(ConnectionTypeModelProperty.KEY, new ConnectionTypeModelProperty(parsedParameter.getType().getRawType()));
             }
+
+            UseConfig useConfig = parsedParameter.getAnnotation(UseConfig.class);
+            if (useConfig != null)
+            {
+                operation.withModelProperty(UseConfigTypeModelProperty.KEY, new UseConfigTypeModelProperty(parsedParameter.getType().getRawType()));
+            }
+        }
+    }
+
+    private void checkAnnotationIsNotUsedMoreThanOnce(Method method, Class annotationClass)
+    {
+        Stream<java.lang.reflect.Parameter> parametersStream = Arrays
```

Comment:
you don't need to turn the array into a list and back to a stream. the Streams class contains a method to get a stream out of an array